.npmignore

```diff
@@ -16,4 +16,7 @@ book.json
 NOTICE
 webpack.dev.config.js
 webpack.prod.config.js
-esp-js-*.tgz
\ No newline at end of file
+esp-js-*.tgz
+.babelrc
+jest.config
+yarn.lock
```

Comment:
This is in the exclude file for the npm packge so `.babelrc` and `jest.config` don't need to be there. 

I'll have to check regarding `yarn.lock`